src/views/docs-view/components/product-docs-search/index.tsx

```diff
@@ -54,7 +59,7 @@ export default function ProductDocsSearch() {
                     query,
                     params: {
                       clickAnalytics: true,
-                      hitsPerPage: 25,
+                      hitsPerPage: 20,
```

Comment:
The search insights event can only handle 20 results for hits viewed, so limiting this accordingly.